pootle/checks.py

```diff
@@ -114,8 +114,8 @@ def check_redis(app_configs=None, **kwargs):
         workers = Worker.all(queue.connection)
     except Exception as e:
         conn_settings = queue.connection.connection_pool.connection_kwargs
-        errors.append(checks.Critical(_("Could not connect to Redis (%s)") % (e),
-            hint=_("Make sure Redis is running on %(host)s:%(port)s") % (conn_settings),
+        errors.append(checks.Critical(_("Could not connect to Redis (%s)", e),
+            hint=_("Make sure Redis is running on %(host)s:%(port)s") % conn_settings,
```

Comment:
It's correct. ugettext doesn't support dict formatting. cf
https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#standard-translation